src/common/AdaptiveStream.cpp

```diff
@@ -1006,10 +1006,6 @@ bool AdaptiveStream::seek_time(double seek_seconds, bool preceeding, bool& needR
   if (!current_rep_)
     return false;
 
-  if (state_ == STOPPED)
-    // For subtitles which come in one file we should return true!
-    return current_rep_->segments_.empty();
```

Comment:
Works fine for single file but our state becomes 'STOPPED' once we're in the last segment of a multi-segment period. Since subtitle stream segments are quite long (several minutes usually) the effect of the code here was seeking once in this segment would never be able to recover the sub stream - subs are now off until we stop/start playback or for multiperiod you would have to exit/re-enter the period.
For audio/video streams, trying to seek once already in the last segment would stop playback.
Removing this allows for the appropriate things to happen to 'restart' the stopped stream (other changes help address this also)